xskillscore/core/comparative.py

```diff
@@ -167,3 +177,141 @@ def _categorical_metric(observations, forecasts, dim):
     walk.coords["alpha"] = alpha
     walk = _add_as_coord(walk, confidence, "confidence")
     return walk
+
+
+def mae_test(
+    forecasts1,
+    forecasts2,
+    observations=None,
+    dim=[],
+    time_dim="time",
+    alpha=0.05,
+):
+    """
+    Returns the Jolliffe and Ebert MAE significance test whether forecasts1 and
+    forecasts2 have different mean absolute error (MAE) at significance level alpha.
+    https://www.cawcr.gov.au/projects/verification/CIdiff/FAQ-CIdiff.html
+
+
+    .. note::
+        ``alpha`` is the desired significance level and the maximum acceptable risk of
+        falsely rejecting the null-hypothesis. The smaller the value of α the greater
+        the strength of the test. The confidence level of the test is defined as
+        1 - alpha, and often expressed as a percentage. So for example a significance
+        level of 0.05, is equivalent to a 95% confidence level.
+        Source: NIST/SEMATECH e-Handbook of Statistical Methods.
+        https://www.itl.nist.gov/div898/handbook/prc/section1/prc14.htm
+
+    Parameters
+    ----------
+    forecasts1 : xarray.Dataset or xarray.DataArray
+        forecasts1 to be compared to observations
```

Comment:
I feel like this is a circular definition. Can you just say `First forecast to be compared to the observations` and `Second` for the other?